pkg/operation/botanist/vpnshoot.go

```diff
@@ -26,22 +26,13 @@ import (
 
 // DefaultVPNShoot returns a deployer for the VPNShoot
 func (b *Botanist) DefaultVPNShoot() (vpnshoot.Interface, error) {
-	var (
-		imageName         = images.ImageNameVpnShoot
```

Comment:
I assume we can drop https://github.com/gardener/gardener/blob/b53d99f3edadcfa6d1fb66510debbf5777a92209/charts/images.yaml#L136-L139 now?

Reply:
done